app/assets/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -76,6 +78,15 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
     this.props.onChange(propertyName, parseInt(value));
   };
 
+  onChangeRenderMissingDataBlack = (value: boolean): void => {
+    Toast.warning(
```

Comment:
I didn't manage to read the longer of the two toast messages before it disappeared. I would double the amount of display time for the toast.